src/Culling/babylon.boundingSphere.ts

```diff
@@ -105,12 +105,29 @@ module BABYLON {
          * @returns true if there is an intersection
          */
         public isInFrustum(frustumPlanes: Array<DeepImmutable<Plane>>): boolean {
-            for (var i = 0; i < 6; i++) {
-                if (frustumPlanes[i].dotCoordinate(this.centerWorld) <= -this.radiusWorld) {
+            let center = this.centerWorld;
+            let radius = this.radiusWorld;
+            for (let i = 0; i < 6; i++) {
+                if (frustumPlanes[i].dotCoordinate(center) <= -radius) {
                     return false;
                 }
             }
+            return true;
+        }
 
+        /**
+         * Tests if the bounding sphere center is inbetween the frustum planes.

```

Comment:
typo
```suggestion
         * Tests if the bounding sphere center is in between the frustum planes.
```

Reply:
mmhh... the right english spelling is really the adverb "inbetween"

Reply:
oops, no I'm wrong and you're right : https://en.wiktionary.org/wiki/inbetween

Reply:
:)

Reply:
rhhaa beurre d'aile, je l'ai ratée celle-ci
mékékon